root/usr/share/nethesis/nethserver-ns8-migration/apps/nethserver-mattermost/migrate

```diff
@@ -60,6 +60,47 @@ fi
 /sbin/e-smith/config setprop mattermost status disabled || :
 systemctl disable --now mattermost || :
 
+# Setup migration page
+mkdir -p ./webroot
+cat - >webroot/index.html <<EOF
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Mattermost Migration</title>
+  </head>
+  <body>
+    <h1>Mattermost Migration</h1>
+    <p>Site has moved, wait for the DNS update.</a></p>
+  </body>
+</html>
+EOF
+
+mkdir -p /etc/e-smith/templates-custom/etc/httpd/conf.d/zz_mattermost.conf/
```

Comment:
Yes but it's not possible with current template implementation because nextcloud can use a virtual host.

I honestly do not want to maintain another complex template.
If the template-custom approach is not good, we can just leave the app unreachable.

Reply:
:ok_hand: